dobi.sh

```diff
@@ -13,7 +13,7 @@ if [ -z ${1} ]; then
 fi
 
 # load generated version infos
-source gen/gitversion/env/gitversion.env
```

Comment:
Would it hurt to source both env files?
Bb-buildingblock will source both, because it cannot tell which envs the user chooses to use.

Reply:
Done